### ISSUE TITLE
Cannot open new tab in Opera with GroupSpeedDial enabled

### ISSUE THREAD

**Issue by Locutus64** [fastaddons/GroupSpeedDial#208]
I noticed yesterday day that if I click the + on the tabbar in Opera no new tab will be opened, if I rightclick a existing tab and chose open new tab no new tab will open. Today I disabled all my extensions and reenabled and tested and the only time I could not get a new tab was when I Reenabled GroupSpeedDial. Can you please see if you get the same issue with Opera, and if so maybe see if you can figure what in GroupSpeedDial is causing the issue?  Thanks

OS's: Arch, Garuda, Manjaro, Xero - all with the Plasma desktop - all fully updated
Opera 93.0.4585.37

**Comment by Locutus64** [comment]
Nothing on this?

**Comment by fastaddons** [comment]
Hello,
I'm sorry for the delay. I've been dealing with some urgent server issues which took all of my time...
Regarding the problem - my addon cannot override new tab page in Opera, they don't allow it.
So when you open new tab, you should see default Opera new tab page.
You are probably using some special addon that can "detect" opening new tab page and replace it with custom URL - of my addon maybe?
Check the addons you have installed for some "new tab override" or something like that.

**Comment by Locutus64** [comment]
OK I disabled everything except your extension and click the + and then rightclicked a tab and clicked new tab and both times new tabs were created. I tracked the issue to New Tab Start Page. It's default url wasn't working. I switch it to startup usl for your extension which it should of been and now clicking the + or rightclicking and selecting new tab works again. Thanks for the suggestions and help.
